src/quick-lint-js/error.h

```diff
@@ -1194,6 +1194,13 @@
       .error(QLJS_TRANSLATABLE("continue can only be used inside of a loop"),  \
              continue_statement))                                              \
                                                                                \
+  QLJS_ERROR_TYPE(                                                             \
+      error_typescript_style_const_field, "E165",                              \
```

Comment:
Nit: Sort alphabetically by error_ struct name.

Reply:
Resolved in https://github.com/quick-lint/quick-lint-js/pull/376/commits/57e41362fc64be22c3e1b2bfcc1a0b6717c5858c